src/CloudKit/Enums.cs

```diff
@@ -14,7 +14,9 @@ public enum CKAccountStatus : long {
 		CouldNotDetermine = 0,
 		Available = 1,
 		Restricted = 2,
-		NoAccount = 3
+		NoAccount = 3,
+		[Mac(12,0), iOS(15,0), TV(15,0), MacCatalyst(15,0)]
```

Comment:
minor: space before `(`

but that looks like an error enum... and those, being output only, do not need availability